src/index.js

```diff
@@ -92,7 +92,7 @@ async function writePackageJSON(options) {
 async function installDeps() {
   const command = hasYarn() ? 'yarn' : 'npm';
   const args = ['install'];
-  await crossSpawn(command, args, { stdio: 'inherit' });
+  await pify(crossSpawn)(command, args, { stdio: 'inherit' });
```

Comment:
Realized that the `await` in `bin.js` wasn't actually waiting for `installDeps` to finish.